### PR TITLE
新增 torchvision 和 paddle.vision 的对应关系（第二批）

### PR DESCRIPTION
**Pull request by guozixu2001**
1. 新增 16 个 torchvision 和 paddle.vision 的对应关系
2. 修复  `CIFAR10`  `MNIST`   `normalize`  的对应关系中的问题

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference_third_party/torchvision/torchvision.io.read_file.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## [参数完全一致]torchvision.io.read_file
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference_third_party/torchvision/torchvision.io.read_file.md]: 仅参数名不一致 还是 参数完全一致

File: docs/guides/model_convert/convert_from_pytorch/api_difference_third_party/torchvision/torchvision.ops.DeformConv2d.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+| --------------------------- | ------------------------------ | --------------------- |
+| in_channels      | in_channels            | 输入通道数。|
+| out_channels     | out_channels        | 由卷积操作产生的输出的通道数。                |
+| kernel_size      | kernel_size            | 卷积核大小。|
+| stride           | stride               | 步长大小。       |
+| padding          | padding              | 填充大小。       |
+| dilation         | dilation             | 空洞大小。       |
+| -               | deformable_groups    | 可变形卷积组数，PyTorch 无此参数，Paddle 保持默认即可。       |
+| -               | groups               | 三维卷积层的组数，PyTorch 无此参数，Paddle 保持默认即可。       |
+| bias            | -                    | 可变形卷积偏置项，Paddle 无此参数，暂无转写方式。      |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference_third_party/torchvision/torchvision.ops.DeformConv2d.md]: 这个可以对上bias_attr，目前很多是对应到bias_attr，因为paddle的bias_attr支持bool

File: docs/guides/model_convert/convert_from_pytorch/api_difference_third_party/torchvision/torchvision.ops.DeformConv2d.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+torchvision.ops.DeformConv2d(in_channels: int, out_channels: int, kernel_size: int, stride: int = 1, padding: int = 0, dilation: int = 1, groups: int = 1, bias: bool = True)
+```
+
+### [paddle.vision.ops.DeformConv2D](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/vision/transforms/pad_cn.html)
+
+```python
+paddle.vision.ops.DeformConv2D(in_channels, out_channels, kernel_size, stride=1, padding=0, dilation=1, deformable_groups=1, groups=1, weight_attr=None, bias_attr=None)
+```
+
+功能一致，参数完全一致，具体如下：
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference_third_party/torchvision/torchvision.ops.DeformConv2d.md]: 这里和分类不是对应的

File: docs/guides/model_convert/convert_from_pytorch/api_difference_third_party/torchvision/torchvision.ops.ps_roi_pool.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+| torchvision                           | PaddlePaddle       | 备注      |
+| ------------------------------------- | ------------------ | -------- |
+| input                                 | x                  | 输入特征图，仅参数名不一致。|
+| boxes                                 | boxes, boxes_num   | 待执行池化的 ROIs 的框坐标，Paddle 的 boxes 只接受一个形状为 (num_rois, 4) 的 Tensor，用 boxes_num 指定每个样本中的 ROI 数量；PyTorch 的 boxes 可以是形状为 (K, 5) 的 Tensor，其中第一列是批次索引，其余四列是 [x1, y1, x2, y2]；也可以是一个 Tensor List，每个 Tensor 形状为 (L, 4)，表示每个样本中的所有 ROI 坐标，需要转写。|
+| output_size                           | output_size        | 池化后输出的尺寸。|
+| spatial_scale                         | spatial_scale      | 空间比例因子。|
+
+### 转写示例
+#### boxes：待执行池化的 ROIs 的框坐标
+boxes 是 Tensor[K, 5]，其中第一列是批次索引，其余四列是 [x1, y1, x2, y2]。
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference_third_party/torchvision/torchvision.ops.ps_roi_pool.md]: 这里描述能否再优化下，看得不是很直观

File: docs/guides/model_convert/convert_from_pytorch/api_difference_third_party/torchvision/torchvision.ops.ps_roi_pool.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+```python
+# PyTorch 写法
+boxes = torch.tensor([[0, 4, 4, 7, 7], [1, 5, 5, 10, 10]], dtype=torch.float32)
+output = torchvision.ops.ps_roi_pool(input, boxes=boxes, output_size=7, spatial_scale=1.0)
+
+# Paddle 写法
+boxes = paddle.to_tensor([[4, 4, 7, 7], [5, 5, 10, 10]], dtype='float32')
+boxes_num = paddle.to_tensor([1, 1], dtype='int32')
+output = paddle.vision.ops.psroi_pool(x, boxes=boxes, boxes_num=boxes_num, output_size=7, spatial_scale=1.0)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference_third_party/torchvision/torchvision.ops.ps_roi_pool.md:51]: 转写能否再精简下，只需要关键差异行对比即可，最好一眼能直观看出差异

File: docs/guides/model_convert/convert_from_pytorch/api_difference_third_party/torchvision/torchvision.ops.roi_align.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+| ------------------------------------- | ------------------ | -------- |
+| input                                 | x                  | 输入特征图，仅参数名不一致。|
+| boxes                                 | boxes, boxes_num   | 待执行池化的 ROIs 的框坐标，Paddle 的 boxes 只接受一个形状为 (num_rois, 4) 的 Tensor，用 boxes_num 指定每个样本中的 ROI 数量；PyTorch 的 boxes 可以是形状为 (K, 5) 的 Tensor，其中第一列是批次索引，其余四列是 [x1, y1, x2, y2]；也可以是一个 Tensor List，每个 Tensor 形状为 (L, 4)，表示每个样本中的所有 ROI 坐标，需要转写。|
+| output_size                           | output_size        | 池化后输出的尺寸。|
+| spatial_scale                         | spatial_scale      | 空间比例因子。|
+| sampling_ratio                        | sampling_ratio     | 用于计算每个池化输出条柱的输出值的采样点数。|
+| aligned                               | aligned            | 像素移动框是否将其坐标移动-0.5。|
+
+### 转写示例
+#### boxes：待执行池化的 ROIs 的框坐标
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference_third_party/torchvision/torchvision.ops.roi_align.md:43]: 同上

File: docs/guides/model_convert/convert_from_pytorch/api_difference_third_party/torchvision/torchvision.ops.roi_pool.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+| torchvision                           | PaddlePaddle       | 备注      |
+| ------------------------------------- | ------------------ | -------- |
+| input                                 | x                  | 输入特征图，仅参数名不一致。|
+| boxes                                 | boxes, boxes_num   | 待执行池化的 ROIs 的框坐标，Paddle 的 boxes 只接受一个形状为 (num_rois, 4) 的 Tensor，用 boxes_num 指定每个样本中的 ROI 数量；PyTorch 的 boxes 可以是形状为 (K, 5) 的 Tensor，其中第一列是批次索引，其余四列是 [x1, y1, x2, y2]；也可以是一个 Tensor List，每个 Tensor 形状为 (L, 4)，表示每个样本中的所有 ROI 坐标，需要转写。|
+| output_size                           | output_size        | 池化后输出的尺寸。|
+| spatial_scale                         | spatial_scale      | 空间比例因子。|
+
+### 转写示例
+#### boxes：待执行池化的 ROIs 的框坐标
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference_third_party/torchvision/torchvision.ops.roi_pool.md:41]: 同上

File: docs/guides/model_convert/convert_from_pytorch/api_difference_third_party/torchvision/torchvision.ops.ps_roi_pool.md (hunk 5 above)
Comment (guozixu2001) [reply docs/guides/model_convert/convert_from_pytorch/api_difference_third_party/torchvision/torchvision.ops.ps_roi_pool.md:51]: 感觉这里的转写不太能精简了，因为这里转写的参数 `boxes` 和 `boxes_num` 需要一起看方便一点。如果写成下面一行的样子的话，感觉更不直观了呀~
```
# PyTorch 写法
torchvision.ops.ps_roi_pool(input, boxes=torch.tensor([[0, 4, 4, 7, 7], [1, 5, 5, 10, 10]], dtype=torch.float32), output_size=7, spatial_scale=1.0)

# Paddle 写法
paddle.vision.ops.psroi_pool(x, boxes=paddle.to_tensor([[4, 4, 7, 7], [5, 5, 10, 10]], dtype='float32'), boxes_num=paddle.to_tensor([1, 1], dtype='int32'), output_size=7, spatial_scale=1.0)
```

**Review by zhwesky2010 — APPROVED** [review]
LGTM

文档暂时没问题，后续根据文档来实现Matcher，**如果实现Matcher过程中发现文档有问题，及时再提PR更正同步过来**
